fpdf/drawing.py

```diff
@@ -0,0 +1,1023 @@
+from contextlib import contextmanager
+import enum
+import math
+
+
+def _clamp8(value):
+    return 0.0 if value < 0.0 else 255.0 if value > 255.0 else value
+
+
+def _render_number(number):
```

Comment:
What about `float2str` in terms of naming?
Just a suggestion

Reply:
I'm not attached to any particular name, I think I reused `render` for the name of this because I was using it on almost everything else in here. I do agree that `float2str` is more descriptive of what it actually does.

Reply:
I ended up landing on `number_to_str` because I am trying to support the built-in `decimal.Decimal` class for better precision under some circumstances, and it does also handle integers (though Python does generally treat those as a subset of floats).